.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[ISSUE]"
+labels: question
+assignees: Lagovas
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Acra configuration files**
+For AcraServer: 
+    - [ ] configuration file or CLI params that you use to start AcraServer; 
+    - [ ] `encryptor_config.yaml` if used.
+For AcraTranslator: 
+    - [ ] configuration file or CLI params that you use to start AcraTranslator.
+
+**Environment (please complete the following information):**
+- Acra version: [e.g. 0.90.0]
```

Comment:
would be useful to specify database type and version. PostgreSQL 11, MySQL 7, mariadb 10, or some cloud based with supported wire protocol.

Reply:
good idea, done